lib/utils.js

```diff
@@ -199,6 +199,9 @@ exports.encode = function encode (value) {
   else if (typeof value === 'string') {
     return '"' + exports.escape(value) + '"';
   }
+  else if (value instanceof Date) {
+    return value.getTime();
```

Comment:
@phpnode, I've been reading a bit more about dates in OrientDB and I think it's even a more complicated than what you suggest. That format is the default DB format, but the user can change it to something else (e.g. `ALTER DATABASE DATETIMEFORMAT yyyy-MM-dd'T'HH:mm:ss.SSS`). Also, I've tried using the given format in schemaless mode and it got saved as a string.

Given all this, plus considering timezones, I think the only safe way to encode a date is by using the SQL `DATE()` function like this:
```
return 'date("' + getOrientDbUTCDate(value) + '", "yyyy-MM-dd HH:mm:ss", "UTC")';
```

```
function getOrientDbUTCDate(date){
  var yyyy = pad(date.getUTCFullYear(), 4);
  var MM = pad(date.getUTCMonth() + 1, 2);
  var dd = pad(date.getUTCDate(), 2);
  var HH = pad(date.getUTCHours(), 2);
  var mm = pad(date.getUTCMinutes(), 2);
  var ss = pad(date.getUTCSeconds(), 2);
  
  return yyyy + '-' + MM + '-' + dd + ' ' + HH + ':' + mm + ':' + ss;
}

function pad (number, size){
  return (1e4 + number + "").slice(-size);
}
```

What do you think?